events/handlers.js

```diff
@@ -1,194 +1,138 @@
-    fs.open(filePath, "a", 666, function (e, id) {
-      // convert JSON array to CSV string
-      converter.json2csv(
-        req.body,
-        (err, csv) => {
-          if (err) {
-            res.status(500).send();
-            return;
-          }
-          const row = "\n" + csv;
-          fs.write(id, row, null, "utf8", function () {
-            fs.close(id, function () {
-              res.status(200).send();
-            });
-          });
-        },
-        { prependHeader: false }
-      );
+  db.eventModel
+    .create(req.body)
+    .then((event) => {
+      res.status(200).send(event);
+    })
+    .catch(function (e) {
+      res.status(500).send({
+        message: "something went wrong",
+        error: e,
+      });
     });
-  });
 }
 
 function updateEvent(req, res) {
-  lockEvents(req, res, (req, res) => {
-    let results = [];
-    let found = false;
-    fs.createReadStream(filePath)
-      .pipe(csv())
-      .on("data", (data) => {
-        if (data.id == req.params.eventId) {
-          found = true;
-          results.push(req.body);
-        } else {
-          results.push(data);
-        }
-      })
-      .on("end", () => {
-        if (found === false) {
-          res.status(404).send("event not found");
-          return;
-        }
-        const csvWriter = createCsvWriter({
-          path: filePath,
-          header: headerFile,
-        });
-        csvWriter.writeRecords(results).then(() => {
-          res.status(200).send();
-        });
-      })
-      .on("error", (err) => {
-        console.log("error while reading the file", err);
-        res.status(500).send("internal error");
+  db.eventModel
+    .update(req.body, {
+      where: { id: req.params.eventId },
+    })
+    .then(() => {
+      res.status(200).send("ok");
+    })
+    .catch(function (e) {
+      res.status(500).send({
+        message: "something went wrong",
+        error: e,
       });
-  });
+    });
 }
 
 function deleteEvent(req, res) {
-  lockEvents(req, res, (req, res) => {
-    let results = [];
-    let found = false;
-    fs.createReadStream(filePath)
-      .pipe(csv())
-      .on("data", (data) => {
-        if (data.id == req.params.eventId) {
-          found = true;
-          return;
-        }
-        results.push(data);
-      })
-      .on("end", () => {
-        if (found === false) {
-          res.status(404).send("event not found");
-          return;
-        }
-        const csvWriter = createCsvWriter({
-          path: filePath,
-          header: headerFile,
-        });
-        csvWriter.writeRecords(results).then(() => {
-          console.log("The CSV file was written successfully");
-          res.status(200).send();
-        });
-      })
-      .on("error", (err) => {
-        console.log("error while reading the file", err);
-        res.status(500).send("internal error");
+  db.eventModel
+    .destroy({
+      where: { id: req.params.eventId },
```

Comment:
`req.params` values are string, `id` in db are declared as `INTEGER`. This works as long as you provide valid ids in path. Else it would result in unexpected errors. You need to provide some simple validation for `req.params` everywhere you use them.